app/helpers/metrics_helper.rb

```diff
@@ -12,6 +12,40 @@ def sum_rollups(rollups)
     rollups.sum(&:count)
   end
 
+  def parse_trend(trend)
+    return if trend.nil?
```

Comment:
for the view, i use this helper to determine the visuals. for `trending_flat` i've arbitrarily chosen 5%, but could easily change it to 2 or 3% or whichever number we decide on makes sense.

Reply:
@radical-ube 5% seems a good place to start.

Reply:
i wonder about this at various tiers though. like a podcast going from 2000 to 2100 (5% increase) maybe feels kinda small and not too significant. but 20000 to 21000 (still 5%) might be a bit more notable. so are there certain threshholds, maybe based on the size of the podcast or some other characteristics, where we do want to highlight, "hey, your latest episode is trending (up or down) much larger than usual" that might not be solely based on the percentage?

or should it be equal across all podcasts?